### ISSUE TITLE
How to disable automatic update of the SE to prevent bricking of modded save files

### ISSUE THREAD

**Issue by asdgif** [Norbyte/bg3se#400]
Hey, it really bothers me that I can't seem to disable the automatic updating no matter what I attempt to do.
How do I go about disabling this so I can use v15 with my older save rather than v16?
Everytime I launch it, even an older version of the .dll will always update no matter what I try..

**Comment by hawk5005** [comment]
Delete DWRite.dll - that's the updater.

**Comment by asdgif** [comment]
> Delete DWRite.dll - that's the updater.

Right, but when I do BG3MM says that it is missing the script extender but it is in appdata.
I don't think it has the proper flags to look if it actually is there in appdata as "BG3ScriptExtender", why would dwrite.dll specifically have to be in bin if the script extender is all that matters anyways?
I'm trying to troubleshoot my load CTD at 100% and the script extender is the only thing I cannot revert back to when my save worked originally.

Unless this is just a false flag claiming that I'm missing the script extender, but I do not see the version on the lower lefthand side in the main menu, I can only assume it hasn't properly hooked or being detected.

**Comment by asdgif** [comment]
Also, even when I do delete DWRite.dll it manages to put itself back in the /bin folder and then automatically puts the latest version of the script extender in my appdata when I boot up BG3MM. Lol

From my testing, it borks a bunch of mods too.
An autoupdater is the absolute last thing you want in a modded playthrough so how can I go about permentantly disabling the functionality of this?

**Comment by Kara-bit** [comment]
> BG3MM > Settings > Open preferences > Updater > Select Version xx.x.x Disable Updates X

There's no "updater" there for me

**Comment by Norbyte** [comment]
There should be:
![image](https://github.com/user-attachments/assets/a15606de-3871-4239-b849-04b3a5c5320c)

**Comment by Kara-bit** [comment]
![image](https://github.com/user-attachments/assets/ede59b0b-710c-429a-ae83-f9c8fa661468)
There isn't for me.... (latest release)

**Comment by AngryGayNoises** [comment]
Yeah, exact same issue here: the auto update just bricked a 40 hour save and I don't see any updater options in the mod manager.
